src/auth/auth.service.ts

```
@@ -20,15 +20,18 @@ import { addHours, addDays, addMinutes } from 'date-fns';
+import type { LoginDto } from './dto/login-user.dto';
 import { LogInProvider } from './providers/loginProvider';
 import { JwtPayload } from './interfaces/jwt-payload.interface';
 import { TeamService } from './services/team.service';
-import { SkillVerification, VerificationStatus } from './entities/skills-verification.entity';
-import type { 
-  CreateSkillVerificationDto, 
-  SkillAssessmentDto, 
-  UpdateSkillVerificationDto 
+import {
+  SkillVerification,
+  VerificationStatus,
```

ESLint

[error] 'VerificationStatus' is defined but never used.

(@typescript-eslint/no-unused-vars)